keywords/couchbaseserver.py

```diff
@@ -199,10 +198,28 @@ def delete_buckets(self):
             resp_obj = resp.json()
             if "status" not in resp_obj:
                 break
-            else:
-                count += 1
+            count += 1
             time.sleep(60)
 
+        query_url = self.url.replace("8091", "8093")
+        del_pdstmt_query_data = {"statement": "delete from system:prepareds"}
+        verify_pdstmt_query_data = {"statement": "select * from system:prepareds"}
+        resp = self._session.post("{}/query/service".format(query_url), data=del_pdstmt_query_data)
+        resp_obj = resp.json()
+
+        count = 0
+        while count < 5:
+            resp = self._session.post("{}/query/service".format(query_url), data=verify_pdstmt_query_data)
+            resp_obj = resp.json()
+            status = resp_obj["status"]
+            result_count = resp_obj["metrics"]["resultCount"]
+            if status == "success":
+                if result_count == 0:
+                    break
+                count += 1
+                print("counting one more time for prepare statement check", count)
```

Comment:
Can we import the logger for this

Reply:
forgot to remove the print statement. Removed it now.